src/components/MessageSender.js

```diff
@@ -1,7 +1,57 @@
 import React from 'react';
 import styled from 'styled-components';
+import ChattingScreen from './ChattingScreen';
 
-export default function MessageSender() {
-	return <Wrapper>메시지 전송부</Wrapper>;
+export default function MessageSender({
+  addNewChat,
+  newMessage,
+  changeInputChat,
+}) {
+  return (
+    <Wrapper>
+      <Form>
+        <Input
+          type="text"
+          name="content"
+          value={newMessage.content}
+          onChange={changeInputChat}
+        ></Input>
+        <Button onClick={addNewChat}>전송</Button>
+      </Form>
+    </Wrapper>
+  );
 }
-const Wrapper = styled.div``;
+const Wrapper = styled.div`
+  position: fixed;
+  bottom: 0px;
+  left: 0px;
+  right: 0px;
```

Comment:
left :0 right:0 값으로 위치조정해서 화면에 꽉차게 하고 싶으셨던거라면 이렇게 해도 좋을 것 같아요

```suggestion
 width:100%;
```

Reply:
오 감사합니다!